flexmeasures/ui/utils/color_defaults.py

```diff
@@ -0,0 +1,51 @@
+def get_color_settings(account):
```

Comment:
Here (and in some other util functions) we can add some type hints - here it would be `account: Account` and return type: `-> dict[str, str]`